projects/kit/components/input-date-time/input-date-time.component.ts

```diff
@@ -314,10 +316,23 @@ export class TuiInputDateTimeComponent
         min: TuiDay,
         max: TuiDay,
         timeMode: TuiTimeMode,
+        dateFormat: TuiDateMode,
     ): TuiTextMaskOptions {
         return {
-            mask: [...TUI_DATE_MASK, ',', ' ', ...tuiCreateTimeMask(timeMode)],
-            pipe: tuiCreateAutoCorrectedDateTimePipe({value: day, min, max}, timeMode),
+            // TODO finish localization in {@link https://github.com/TinkoffCreditSystems/taiga-ui/issues/954 issue}
+            mask: [
+                ...tuiCreateDateMask('DMY', '.'),
+                ',',
+                ' ',
+                ...tuiCreateTimeMask(timeMode),
+            ],
```

Comment:
What do you think about? We could move this code elsewhere. And indicate that in the future the internal implementation will change.

```ts
mask: tuiInternalCreateMask(timeMode)
```

```ts
/**
 * @note: TODO finish localization in {@link https://github.com/TinkoffCreditSystems/taiga-ui/issues/954 issue}
**/
export function tuiInternalCreateMask(timeMode) {
  return [
      ...tuiCreateDateMask('DMY', '.'),
      ',',
      ' ',
      ...tuiCreateTimeMask(timeMode),
   ];
}
```